src/main/kotlin/org/rust/ide/utils/import/ImportCandidatesCollector.kt

```diff
@@ -129,6 +137,14 @@ object ImportCandidatesCollector {
             .flatMap { it.withModuleReexports(project).asSequence() }
             .mapNotNull { importItem -> importItem.toImportCandidate(superMods) }
             .filterImportCandidates(attributes)
+            // make std appear first
+            .sortedBy {
+                if (it.qualifiedNamedItem.containingCrate?.origin == PackageOrigin.STDLIB) {
+                    "0${it.qualifiedNamedItem}"
+                } else {
+                    "1${it.qualifiedNamedItem}"
+                }
+            }
```

Comment:
Exactly the same code as above. Let's create single comparator for `ImportCandidate` or implement `Comparable` interface for `ImportCandidate`